### DISCUSSION TITLE
Duda complejidad temporal

### DISCUSSION THREAD

**Discussion by SantiagoBravo05** [INGCOM-UNRN-P1/cursada-2023#219] [Preguntas y respuestas] [answered]
Tengo un codigo con un triple bucle anidado, seria un
O(n^2^2)   ?
o un
O(n^3)       ?

**Comment by luciano-repetti** [comment, marked as answer]
Buenas! La notación en Big O tendría que ser O(n^3) que es lo mismo a O(n*n*n) los tres bucles anidados, uno dentro de otro.
